reference/performances_guide/index.rst

```diff
@@ -40,3 +56,192 @@ Here is the complete formula to check if you have, by far, more indexes than the
+
+We've tested performances with a set of 10k attributes whose 500 attributes usable in the grids
+(10k attributes in total not 10k attributes per product).
+
+.. warning::
+
+    Screens impacted are the following, we're releasing performances fixes as 1.4 patches:
+      - **[fixed in 1.4.12]** (PIM-5208) the product grid loading due to manage filters design (Community Edition)
+      - **[fixed in 1.4.12]** (PIM-5208) the variant product grid loading due to manage filters design (Community Edition)
+
+    **If you plan to use the PIM with more than 500 attributes usable in the grids, please contact us.**
+
+More than 10k families?
+-----------------------
+
+The number of families will impact performances of the PIM in several ways.
+We've tested performances with a set of 10k families in total.
+
+.. warning::
+
+    On this axis, we still have different issues. We've tested with 10k families and a low amount of attributes and categories.
+
+    Screens impacted are the following and we're releasing performances fixes as 1.4 patches:
+      - **[fixed in 1.4.12]** (PIM-5194) the edition of a product and change its family (Community Edition)
+      - the product grid loading due to the synchronous loading of the family filter (Community Edition)
+      - the creation of a product (Community Edition)
+      - the configure step of the product / mass edit / change family (Community Edition)
+      - the creation of a channel due to the creation of families requirements (Community Edition)
+
+    **If you plan to use the PIM with more than 10k families, please contact us.**
+
+More than 10k categories?
+-------------------------
+
+The number of categories will impact performances of the PIM in several ways.
+We've tested performances with a set of 10k categories in total.
+
+.. warning::
+
+    On this axis, we still have different issues. We've tested with 10k categories and a low amount of families and attributes.
+
+    Screens impacted are the following and we're releasing performances fixes as 1.4 patches:
+      - change and save permissions on the root category of a large tree (Enterprise Edition)
+
+    **If you plan to use the PIM with more than 10k categories, please contact us.**
+
+Export more than 100k products?
+-------------------------------
+
+We had use cases where customers would export 270k products and had issues with the memory usage.
+
+Most of the massive operations of the PIM as imports and exports are bulked by page of lines (or objects) to avoid too large memory usage.
+
+As each product can have different properties, the export kept the transformed array in memory to add missing columns from a line to another.
+
+In the version 1.4.9 (PIM-5127), we changed the internal behaviour of the CsvProductWriter to use a file buffer to temporary write each transformed array before to aggregate the final result.
+
+So, in a product export, the limitation is not anymore the memory size but the hard drive space.
```

Comment:
As a conclusion, for product export, limitation is no more available memory but available hard drive space.